gateway/webhook/webhookTransformer.go

```diff
@@ -33,19 +35,29 @@ type transformerBatchResponseT struct {
 	statusCode int
 }
 
-func (bt *batchWebhookTransformerT) markRepsonseFail(reason string) transformerResponse {
+func (bt *batchWebhookTransformerT) markResponseFail(sourceType, reason string) transformerResponse {
 	resp := transformerResponse{
 		Err:        response.GetStatus(reason),
 		StatusCode: response.GetErrorStatusCode(reason),
 	}
 	bt.stats.failedStat.Count(1)
+	transformFailedCountStat := stats.Default.NewTaggedStat("source_transform_failed_jobs", stats.CountType, stats.Tags{
```

Comment:
Can this be initialized here as markResponseFail gets invoked for each event failure ?

Reply:
But each time reason can be different right and also this specific to sourceType so we can't initialise like bt.stats.failedStat so we need to create stat every single time.

Reply:
Makes sense
[non-blocking] Can we change it in the following way ?
```suggestion
stats.Default.NewTaggedStat("source_transform_failed_jobs", stats.CountType, stats.Tags{}).Count(1)
```

Reply:
Done